pkg/dao/db.go

```diff
@@ -54,6 +54,7 @@ func (handler *DatabaseHandler) DoInitialSetup() error {
 	}
 	if _, err := handler.DB.Exec(`CREATE UNLOGGED TABLE IF NOT EXISTS mobileappmetrics (
 		clientId char(80) NOT NULL CHECK (clientId <> ''),
+		event_type char(30) NOT NULL CHECK(event_type <> ''),
```

Comment:
@paolobueno I think it is a good idea to make event_type part of the PRIMARY KEY.
Not 100% sure, but I think I was running into some problems when I was running the integration tests in Android SDK because of this. Like, init and security events were processed within the same millisecond.

Reply:
I see, might improve querying times too I guess, same millisecond could also happen in real clients in the future if the metrics SDKs get optimized to batch requests.

Reply:
+1 on query improvement aspect